doozer/doozerlib/cli/release_gen_assembly.py

```diff
@@ -306,6 +319,27 @@ def _validate_params(self):
                     f"Invalid assembly name: {self.gen_assembly_name}. Has to include release field. Eg: 4.18.0-0"
                 )
 
+    def _set_release_date_and_in_flight(self):
+        if self.custom:
+            return
+
+        if not self.release_date:
+            self.logger.info("Release date not provided. Fetching from release schedule...")
+            self.release_date = get_assembly_release_date(
+                self.gen_assembly_name, self.assembly_type, self.runtime.group
+            )
+        self.logger.info("Using release date: %s", self.release_date)
+
+        determined_in_flight = get_in_flight(self.runtime.group, self.release_date)
+        if self.in_flight:
+            if self.in_flight != determined_in_flight:
```

Comment:
if self.in_flight and self.in_flight != determined_in_flight

If humans deliberately set it, we should just honor the human input, then no need to get_in_flight, during ec/rc the get_in_flight may get none and input value in_flight exist, the warning is not correct

Reply:
yes I will add a None check. I can make it an info message